prody2/protocols/protocol_defvec.py

```diff
@@ -69,41 +69,87 @@ def _defineParams(self, form):
                            '(an EM volume converted into pseudoatoms)'
                            'The two structures should have the same number of nodes.')
 
+        form.addSection(label='Animation')        
+        form.addParam('rmsd', FloatParam, default=0,
+                      label='RMSD Amplitude (A)',
+                      help='Used only for animations of computed normal modes. '
+                      'This is the maximal amplitude with which atoms or pseudoatoms are moved '
+                      'along the deformation vector in the animations. \n'
+                      'The default value of 0 means use the actual RMSD between the structures.')
+        form.addParam('n_steps', IntParam, default=10,
+                      expertLevel=LEVEL_ADVANCED,
+                      label='Number of frames',
+                      help='Number of frames used in each direction of animations.')
+        form.addParam('pos', BooleanParam, default=True,
+                      expertLevel=LEVEL_ADVANCED,
+                      label="Include positive direction",
+                      help='Elect whether to animate in the positive direction '
+                           'from mobile to target.')
+        form.addParam('neg', BooleanParam, default=False,
+                      expertLevel=LEVEL_ADVANCED,
+                      label="Include negative direction",
+                      help='Elect whether to animate in the negative direction, '
+                           'extrapolating from mobile further away from target.')
+
     # --------------------------- STEPS functions ------------------------------
     def _insertAllSteps(self):
         self.mobFn = self.mobStructure.get().getFileName()
         self.tarFn = self.tarStructure.get().getFileName()
         self._insertFunctionStep('defvecStep', self.mobFn, self.tarFn)
+        self._insertFunctionStep('animateModesStep', self.n_steps.get(),
+                                 self.neg.get(), self.pos.get())
+        self._insertFunctionStep('computeAtomShiftsStep')
```

Comment:
this probably shouldn´t be here. Also merge with master and resolve conflicts related to it now being there.